en/components/grid.md

```diff
@@ -639,3 +638,16 @@ Our community is active and always welcoming to new ideas.
 
 * [Ignite UI for Angular **Forums**](https://www.infragistics.com/community/forums/f/ignite-ui-for-angular)
 * [Ignite UI for Angular **GitHub**](https://github.com/IgniteUI/igniteui-angular)
+
+## Grid Limitations
+
+<div ckass="devider"></div>
+
+|Limitation|Description|
+|--- |--- |
+|Column widths set in `percentage` and `px`|Currently we do not support mixing of column widths with `%` and `px`.
+|When `height` of the grid's parent element is not set| The `grid` is not rendered correctly when the `height` of the parent container is not set.|
+|When grid's `height` is not set|When grid's `height` is not set, it will span the parent cantainer it is placed in.|
+|`NaN` is returned when try to filter column of type number|if value different than `number` is entered into the filtering input, `NaN` is returned due incorrect cast and no filter is applied.|
+|Grid `width` does not depend on the column widths | The `width` of all columns does not determine the spanning of the grid itself, it is determined by the parent container dimensions or the difined grid's `width`.
```

Comment:
grid itself.  It is determined by the parent container dimensions or the defined grid's `width`.